http/src/main/resources/reference.conf

```diff
@@ -230,7 +230,7 @@ azure {
 
    coa-app-config {
      instrumentation-enabled = false
-     chart-name = "/leonardo/cromwell-on-azure"
+     chart-name = "cromwell-helm/cromwell-on-azure"
```

Comment:
Am confusing myself -- this is just referencing the remote chart (in the repo) right?

If this works then do we need to pull the charts to the filesystem `/leonardo` at all?

E.g. does this work if you just remove the `updateAndPullChart` calls?

Reply:
I guess that it probably does work, but my concern here is handling the caching so we don't pull the chart at every app creation 🤔

Reply:
Maybe `reference.conf` should still have `chart-name = "/leonardo/cromwell-on-azure"` so it uses the cached version.

Reply:
I could try to do the `updateAndPullChart` call using the remote chart, and then doing the install with the local chart? But I don't know if the `updateAndPUllChart` is clever enough to not pull every time if the chart already exist in the destination directory

Reply:
And more complexity about caching (sorry):
* What happens if multiple things concurrently try to pull the chart to the filesystem (e.g. do we need a lock or something)?
* What can trigger invalidating the cached chart and pulling a new chart version? That could be for a separate PR, but that will become important when we can dynamically update chart versions, right?

Reply:
>  I don't know if the updateAndPUllChart is clever enough to not pull every time if the chart already exist in the destination directory

Yeah, not sure either 🤔 

Because caching is hard -- what if we just don't do it for the first pass and use the remote chart every time? These charts are all in GCS I think, and hopefully pretty reliable..

Reply:
Yeah, that's why maybe we could try without caching and see if impacts app creation time too much?

Reply:
Haha looks like we are in agreement :) But that means not even using the new PullChart function 😭

Reply:
Yeah 😿